extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.css

```diff
@@ -68,6 +70,29 @@ amp-story:not([supports-landscape]) + amp-subscriptions-dialog.i-amphtml-story-s
   background: #EDEDED !important;
 }
 
+.i-amphtml-story-subscriptions-dialog-banner-button {
+  height: 100% !important;
+  display: flex !important;
+  justify-content: flex-end !important;
+  align-items: center !important;
```

Comment:
See comment above, remove justify-content and align-items is handled by the container.

Reply:
We actually need still need the align-items: center to center align the arrow and string

Reply:
I see! Thanks for looking into it and explaining :)